### DISCUSSION TITLE
Welcome to cookiecutter Discussions!

### DISCUSSION THREAD

**Discussion by ssbarnea** [cookiecutter/cookiecutter#1554] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by glumia** [comment]
Nice initiative! This could help us to keep the issues backlog clean with only actionable items.

I'll start with the introduction! I personally use cookiecutter to template Python microservices and since a bunch of months I'm one of the mantainers of this little gem :)

I joined the team with the aim of disposing of the pile of issues accumulated in the last years and I hope it's done soon so we can focus on stability improvements and new useful features!
